src/index.html

```diff
@@ -9,6 +9,62 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div
+      class="img">
+        <a
+        href="#">
+          <img src="../src/images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav
+      class="nav">
+
+
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="menu is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
```

Comment:
```suggestion
          </li>
          
          <li class="nav-item">
```

Add empty line between siblings blocks inside same parent block